tests/test_mu.py

```diff
@@ -85,49 +85,6 @@ def make_func(self, **kw):
         self.addCleanup(archive.remove)
         return LambdaFunction(func_data, archive)
 
-    def test_publishes_a_lambda(self):
-        session_factory = self.replay_flight_data("test_publishes_a_lambda")
-        mgr = LambdaManager(session_factory)
-        func = self.make_func()
-        self.addCleanup(mgr.remove, func)
-        result = mgr.publish(func)
-        self.assertEqual(result["CodeSize"], 169)
-
-    def test_publish_a_lambda_with_layer_and_concurrency(self):
-        factory = self.replay_flight_data('test_lambda_layer_concurrent_publish')
-        mgr = LambdaManager(factory)
-        layers = ['arn:aws:lambda:us-east-1:644160558196:layer:CustodianLayer:2']
-        func = self.make_func(
-            concurrency=5,
-            layers=layers)
-        self.addCleanup(mgr.remove, func)
-
-        result = mgr.publish(func)
-        self.assertEqual(result['Layers'][0]['Arn'], layers[0])
-        state = mgr.get(func.name)
-        self.assertEqual(state['Concurrency']['ReservedConcurrentExecutions'], 5)
-
-        func = self.make_func(layers=layers)
-        output = self.capture_logging("custodian.serverless", level=logging.DEBUG)
-        result = mgr.publish(func)
-        self.assertEqual(result['Layers'][0]['Arn'], layers[0])
-
-        lines = output.getvalue().strip().split("\n")
-        self.assertFalse('Updating function: test-foo-bar config Layers' in lines)
-        self.assertTrue('Removing function: test-foo-bar concurrency' in lines)
-
-    def test_can_switch_runtimes(self):
-        session_factory = self.replay_flight_data("test_can_switch_runtimes")
-        func = self.make_func()
-        mgr = LambdaManager(session_factory)
-        self.addCleanup(mgr.remove, func)
-        result = mgr.publish(func)
-        self.assertEqual(result["Runtime"], "python2.7")
-
-        func.func_data["runtime"] = "python3.6"
-        result = mgr.publish(func)
-        self.assertEqual(result["Runtime"], "python3.6")
-
```

Comment:
Moved these tests from the parent `Publish` class to the child `PolicyLambdaProvision`. Some VS Code testing patterns would run the tests twice if they were part of the parent class. Not a huge deal but a bit weird, and the tests probably make more sense on that child class anyway.

Reply:
hmm.. its unfortunate because the intent on these tests was that they were not policy lambda dependent.

Reply:
Ah thanks, could revert the change (since the awkwardness seems specific to vs code) or put it in a separate test class if that's friendlier.